Peaky.Tests/TestExecutionTests.cs

```diff
@@ -121,6 +121,14 @@ public async Task When_a_test_with_a_return_value_throws_then_a_500_Test_Failed_
             response.ShouldFailWith(HttpStatusCode.InternalServerError);
         }
 
+        [Fact]
+        public async Task When_a_test_throws_then_a_503_Test_Failed_is_returned()
```

Comment:
The test name can be more specific, since this isn't the behavior for just any exception.